kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/createWLSDomain.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+#Copyright (c) 2021, Oracle and/or its affiliates.
+#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+#Adopted from https://github.com/oracle/docker-images/blob/main/OracleWebLogic/samples/12213-domain-home-in-image/container-scripts/createWLSDomain.sh
+#Define DOMAIN_HOME
+echo "Domain Home is: " $DOMAIN_HOME
+
+ADD_DOMAIN=1
+if [  -f ${DOMAIN_HOME}/servers/${ADMIN_NAME}/logs/${ADMIN_NAME}.log ]; then
+    exit
+fi
```

Comment:
I suspect ^^^ is intended to be a sanity check to see if the domain was already created - but now that the admin server is no longer launched during image creation, the check is no longer applicable.  How about replacing with some  pre-req checks.  For example:

```
if [ -z "${DOMAIN_HOME}" ]; then
   echo "Error: DOMAIN_HOME not set."
   exit 1
fi

if !  mkdir -p $DOMAIN_HOME ;  then
  echo "Error: Could not create domain directory '${DOMAIN_HOME}'."
  exit 1
fi

if ! touch $DOMAIN_HOME/test-file ; then
  echo "Error: No permission to create a file in domain home '$DOMAIN_HOME'. id=$(id). Parent dir contents:"
  ls -l $(dirname $DOMAIN_HOME)
  exit 1
fi
rm -f test-file

dfiles="$(ls -A $DOMAIN_HOME)" 
if [ ! -z dfiles ]; then
  echo "Error: Domain home '$DOMAIN_HOME' must be empty before we run the domain home creation script."
  echo "$dfiles"
  exit 1
fi
```

Reply:
Thanks. Added except for mkdir -p $DOMAIN_HOME which was already done in dockerfile before invoking this script